config/redis.yml

```diff
@@ -4,6 +4,10 @@ development:
 test:
   host: localhost
   port: 6379
+staging:
+  host: ENV['REDIS_URL']
```

Comment:
I'd leave this localhost?

REDIS_URL is a bit more then the host, it's actually this enitre config. You can provide it with a url key though if you want to use the env var:

You can also specify connection options as a redis:// URL:

`redis = Redis.new(:url => "redis://:p4ssw0rd@10.0.1.1:6380/15")`

see here: https://github.com/redis/redis-rb#getting-started